File: 프로그래머스/0/120818. 옷가게 할인 받기/옷가게 할인 받기.java
class Solution {
    public int solution(int price) {
        if(price >= 500000) return 80 * price / 100;
        if(price >= 300000) return 90 * price / 100;
        if(price >= 100000) return 95 * price / 100;
        return price;
    }
}

File: 프로그래머스/0/120818. 옷가게 할인 받기/옷가게 할인 받기.js
function solution(price) {
    if(price >= 500000) {
        return Math.floor(80 * price / 100);
    }
    if(price >= 300000) {
        return Math.floor(90 * price / 100);
    }
    if(price >= 100000) {
        return Math.floor(95 * price / 100);
    }
    return price;
}